examples/tts/conf/hifigan_dataset/hifigan_22050.yaml

```diff
@@ -19,6 +14,19 @@ log_ds_meta: ???
 
 log_dir: ???
 
+mel_dim: 80
+lowfreq: 0
+highfreq: 8000
+
+# Change these values depending on your sampling rate.
+sample_rate: 22050
+win_length: 1024
+hop_length: 256
+upsample_rates: [8, 8, 2, 2]
+train_n_samples: 8192
+val_min_duration: 3.0
```

Comment:
nit: could you pls add a comment showing the unit is "seconds"?

Reply:
Sure, will change it.